### ISSUE TITLE
React 16.2.0 produces Module not found error

### ISSUE THREAD

**Issue by karland** [nozzle/react-static#235]
This works fine:

```JSON
"react": "16.1.1",
"react-dom": "16.1.1",
```

```bash

yarn run v1.3.2
$ react-static start

=> Copying public directory...
=> [✓] Public directory copied: 97.243ms
=> Building Routes...
=> [✓] Routes Built: 4.951ms
=> Building App Bundle...
=> [✓] Build Complete: 4968.120ms
=> [✓] App serving at http://localhost:3000
```

This produces "Module not found" errors:

```JSON
"react": "16.2.0",
"react-dom": "16.2.0",
```

```bash
yarn run v1.3.2
$ react-static start

=> Copying public directory...
=> [✓] Public directory copied: 443.031ms
=> Building Routes...
=> [✓] Routes Built: 10.215ms
=> Building App Bundle...
=> [✓] App serving at http://localhost:3000
Failed to rebuild.
./node_modules/react-static/lib/index.js
Module not found: Can't resolve 'react-helmet' in 'd:\my\myProject\node_modules\react-static\lib'

./build/react-static-routes.js
Module not found: Can't resolve 'react-router-dom' in 'd:\my\myProject\build'

./node_modules/react-static/lib/index.js
Module not found: Can't resolve 'react-router-dom' in 'd:\my\myProject\node_modules\react-static\lib'
```

Versions:
* Windows: 10-64
* node: 9.2.0
* react-static: 4.5.0

Any ideas? Thanks.

**Comment by tannerlinsley** [comment]
Still experiencing this issue?

**Comment by karland** [comment]
@tannerlinsley Thanks for asking. Yes, indeed, I still have the same problem. I updated all packages yesterday and my working package.json now is:

```JSON
{
  "name": "myProject",
  "version": "0.3.0",
  "main": "index.js",
  "license": "MIT",
  "scripts": {
    "start": "react-static start",
    "build": "react-static build",
    "serve": "serve deploy -p 3001"
  },
  "dependencies": {
    "bowser": "^1.8.1",
    "grid-styled": "^2.0.0-11",
    "lazysizes": "^4.0.1",
    "moment": "^2.20.1",
    "react": "16.1.1",
    "react-dom": "16.1.1",
    "react-router": "^4.2.0",
    "react-static": "^4.7.0-beta.0",
    "styled-components": "^2.3.3",
    "styled-system": "^1.1.1"
  },
  "devDependencies": {
    "eslint": "^4.13.1",
    "eslint-config-react-app": "^2.0.1",
    "eslint-config-react-tools": "1.x.x",
    "eslint-plugin-flowtype": "^2.40.1",
    "eslint-plugin-import": "^2.8.0",
    "eslint-plugin-jsx-a11y": "^6.0.3",
    "eslint-plugin-react": "^7.5.1",
    "prettier-eslint": "^8.4.0",
    "serve": "^6.4.3"
  },
  "resolutions": {
    "styled-system": "1.1.1"
  }
}
```

Changing to 

```JSON
    "react": "16.2.0",
    "react-dom": "16.2.0",
```

results still in the above mentioned error. I just tried it yesterday. If you tell me, that 16.2.0 works fine on your system, I probably have to try to narrow it down. I am not sure what changed from React 16.1.1 to React 16.2.0 that could cause the error or if this is a Windows specific error. Maybe it is a similar issue like this [PR #191](https://github.com/nozzle/react-static/pull/191), but I have difficulties to understand, why it only occurs with React 16.2.0.

**Comment by karland** [comment]
I tried one more thing and included 

```JSON
    "react-helmet": "^5.2.0",
    "react-router-dom": "^4.2.2",
```

directly in my package.json. Now it works, but I find it strange, that it is not working through the dependencies in the package.json of react-static.

**Comment by tannerlinsley** [comment]
I've got this issue replicated on my machine now :) Now I'm working on a fix.  I agree, this one is pretty strange... I'm not sure what is happening just yet.  Let me know if you find any clues!

**Comment by tannerlinsley** [comment]
Fixed with f96e753b6c7dd305ca476b8578dbeb34bc59236c and 457477d6d6d2db79f95c1a6f01d22cfdc0dcefbe
